### PR TITLE
Added upscaling support for images

### PR DESCRIPTION
**Pull request by rgyhuang**
Added a resize function that allows for up-scaling, but maintains aspect ratio. Label kit windows can now be enlarged past the original image size.

### CODE REVIEW

File: streamlit_label_kit/LabelToolKit/__init__.py [hunk 1]
```diff
@@ -94,4 +96,33 @@ def absolute_to_relative(
...
+
+    Returns:
+    PIL.Image: The resized image
+    """
+    original_width, original_height = image.size
+    target_width, target_height = size
+    
+
+    aspect_ratio = original_width / original_height
+    target_aspect_ratio = target_width / target_height
```
Comment (jaharkes) [inline streamlit_label_kit/LabelToolKit/__init__.py]: target_aspect_ratio shouldn't really be relevant. I would probably approach this as getting the original aspect ratio, then based on one dimension (f.i. target_width) computing the new_height with the given aspect ratio, and if that exceeds the target height redoing the computation based on the target_height. I'm sure your code may effectively do the same thing but I find it harder to reason if it catches all corner cases, between scaling up and scaling down, etc.

File: streamlit_label_kit/LabelToolKit/frontend/src/label-tool/Detection.tsx [hunk 2]
```diff
@@ -141,8 +141,8 @@ export const Detection = (args: PythonArgs) => {
     const resizeCanvas = () => {
       const control_width = left_width + right_width;
       const scale_ratio = (window.innerWidth - control_width) / image_size[0];
-      setScale(Math.min(scale_ratio, 1.0))
-      Streamlit.setFrameHeight(image_size[1] * Math.min(scale_ratio, 1.0) + bottom_height);
+      setScale(Math.min(scale_ratio, 1.0));
+      Streamlit.setFrameHeight(image_size[1] * scale_ratio + bottom_height);
```
Comment (jaharkes) [inline streamlit_label_kit/LabelToolKit/frontend/src/label-tool/Detection.tsx]: You didn't change the frameheight in Classification.tsx and Segmentation.tsx, but are changing it in Detection.tsx

Comment (rgyhuang) [reply streamlit_label_kit/LabelToolKit/frontend/src/label-tool/Detection.tsx]: Updated!

**Review by jaharkes — APPROVED** [review]
Lots of unrelated whitespace changes, but it doesn't look like it adds unnecessary whitespace or introduce new problems.

**Pull request merged**